apps/src/blockly/addons/cdoBlockSvg.js

```diff
@@ -117,8 +106,8 @@ export default class BlockSvg extends GoogleBlockly.BlockSvg {
     super.onMouseDown_(e);
   }
 
-  render() {
-    super.render();
+  render(opt_bubble) {
+    super.render(opt_bubble);
```

Comment:
hm weird that git is recognizing this as part of the diff since the other PR was merged. I don't think it should cause any issues?

Reply:
I wonder if you rebase off of the latest from staging if it will still show?

Reply:
I merged staging into this branch that's no longer showing as a diff.